src/validation/rename/find_renaming_pairs.cpp

```diff
@@ -46,10 +46,10 @@ void find_renaming_pairs(Ensemble *ensemble) {
     // We use MW from rename-mam4.
     Real molecular_weight_rename[naerosol_species] = {
         150, 115, 150, 12, 58.5, 135, 250092}; // [kg/kmol]
-    for (int iaero = 0; iaero < naerosol_species; ++iaero) {
-      mass_2_vol[iaero] =
```

Comment:
Isn't `mass_2_vol` used here, though? Without your changes, that is.

Reply:
I wondered that myself, though it's been the inciting error for the last few build crashes. I think the compiler doesn't consider initialization "using" since it never appears on a RHS after initialized.

```
/home/runner/work/mam4xx/mam4xx/src/validation/rename/find_renaming_pairs.cpp:33:10: warning: variable ‘mass_2_vol’ set but not used [-Wunused-but-set-variable]
[76](https://github.com/eagles-project/mam4xx/actions/runs/4481758068/jobs/7878908096#step:7:77)
   33 |     Real mass_2_vol[naerosol_species];
[77](https://github.com/eagles-project/mam4xx/actions/runs/4481758068/jobs/7878908096#step:7:78)
      |          ^~~~~~~~~~
[78](https://github.com/eagles-project/mam4xx/actions/runs/4481758068/jobs/7878908096#step:7:79)
make[2]: *** [src/tests/CMakeFiles/mam4_aging_unit_tests.dir/build.make:76: src/tests/CMakeFiles/mam4_aging_unit_tests.dir/mam4_aging_unit_tests.cpp.o] Terminated
```

Reply:
And it turns out on debug/single it appears to be crashing on the codecov upload

```
[2023-03-21T17:03:19.872Z] ['error'] There was an error running the uploader: Error uploading to [https://codecov.io:](https://codecov.io/) Error: There was an error fetching the storage URL during POST: 500 - Unknown error, please try again later
[108](https://github.com/eagles-project/mam4xx/actions/runs/4481758068/jobs/7878907465#step:13:109)
[2023-03-21T17:03:19.873Z] ['verbose'] The error stack is: Error: Error uploading to [https://codecov.io:](https://codecov.io/) Error: There was an error fetching the storage URL during POST: 500 - Unknown error, please try again later
[109](https://github.com/eagles-project/mam4xx/actions/runs/4481758068/jobs/7878907465#step:13:110)
    at main (/snapshot/repo/dist/src/index.js)
[110](https://github.com/eagles-project/mam4xx/actions/runs/4481758068/jobs/7878907465#step:13:111)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
[111](https://github.com/eagles-project/mam4xx/actions/runs/4481758068/jobs/7878907465#step:13:112)
[2023-03-21T17:03:19.873Z] ['verbose'] End of uploader: 10208 milliseconds
Error: Codecov: Failed to properly upload: The process '/home/runner/work/_actions/codecov/codecov-action/v3/dist/codecov' failed with exit code 255
```

Reply:
Huh. I wonder if there's a race condition between builds. Maybe we should have only one of our builds upload the coverage report. Anyway, thanks for this PR, @mjs271 . I'll approve.